testing/trino-product-tests-launcher/src/test/java/io/trino/tests/product/launcher/cli/TestInvocations.java

```diff
@@ -102,36 +100,10 @@ public static InvocationResult invokeLauncher(String... args)
     {
         Launcher launcher = new Launcher();
         ByteArrayOutputStream out = new ByteArrayOutputStream();
-
-        try (CaptureOutput ignored = new CaptureOutput(out)) {
-            int exitCode = execute(launcher, new Launcher.LauncherBundle(), args);
-            return new InvocationResult(exitCode, out.toString(UTF_8));
-        }
+        int exitCode = execute(launcher, new Launcher.LauncherBundle(), out, args);
+        return new InvocationResult(exitCode, out.toString(UTF_8));
```

Comment:
Can you also update `testDescribeSuite` to match the output exactly, instead of using `contains()`? This is why we haven't noticed the issue, when stdout and stderr were combined and there were additional logger fields included in the output. https://github.com/trinodb/trino/blob/master/.github/bin/build-pt-matrix-from-impacted-connectors.py expects the output to be valid JSON.

Reply:
Done. Now asserting on whole lines